apollo-configservice/src/main/java/com/ctrip/framework/apollo/configservice/ConfigServiceAutoConfiguration.java

```diff
@@ -69,13 +69,18 @@ public GrayReleaseRulesHolder grayReleaseRulesHolder() {
 
   @Bean
   public ConfigService configService() {
+    //四种情况
+    //开启增量更新
+    if (bizConfig.isConfigServiceIncrementalChangeEnabled()) {
+
+    }
+    //开启本地缓存
```

Comment:
Prefer English comments in code.
```suggestion
    // enable local cache
```